test/test_authn.py

```diff
@@ -137,26 +183,25 @@ def test_authn_same_proposal(make_client):
     assert resp.json["user"]["inControl"] == False
 
 
-# # Test against proposal-based authentication only
-# @pytest.mark.parametrize("login_type", ["proposal"], indirect=True)
-# def test_authn_different_proposals(make_client):
-#     """Test two users for different proposals.
-
-#     If a user signs in for a different proposal than an already signed in user,
-#     this user should not be allowed to sign in.
-#     """
-#     client_0 = make_client()
-#     resp = client_0.post(URL_SIGNIN, json=CREDENTIALS_0)
-#     assert resp.status_code == 200
-#     resp = client_0.get(URL_INFO)
-#     assert resp.status_code == 200
-
-#     client_1 = make_client()
-#     resp = client_1.post(URL_SIGNIN, json=CREDENTIALS_1)
-#     assert resp.status_code == 200
-#     import pdb
-#     pdb.set_trace()
-#     assert resp.json["msg"] == "Could not authenticate"
+# Test against proposal-based authentication only
+@pytest.mark.parametrize("login_type", ["proposal"], indirect=True)
+def test_authn_different_proposal(make_client):
+    """Test two users for different proposals.
```

Comment:
So this is not actually testing for different proposals, its testing two users with the same proposal

Reply:
ok, I will cherry pick from develop for this piece